docs/getting-started/server/ef/index.md

```diff
@@ -56,12 +65,18 @@ You can have multiple databases configured and switch between them by changing t
     pwsh migrate.ps1 -postgres
     ```
 
+    :::note
+
+    `pwsh migrate.ps1 -all` will run migrations for all database providers
+
+    :::
```

Comment:
I think we should try and avoid using to many callouts.

```suggestion
    You can also run migrations for all database providers at once using

    ```bash
    pwsh migrate.ps1 -all
    ```
```

Reply:
what's wrong with callouts?

They're a good way to draw attention to some buried fact you want to surface.